Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/1-index](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Custom-Directories](https://hosted.weblate.org/projects/jamulus/custom-directories/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/footertext](https://hosted.weblate.org/projects/jamulus/footertext/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/general](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

pljones commented on _translator-files/po/fr/Contribution.po:
```diff
@@ -59,10 +59,14 @@ msgstr "Si vous pensez qu'il manque de la documentation ou des informations ou q
-msgstr "Si vous souhaitez aider à la traduction, vous devrez disposer d'un compte GitHub et être familier avec l'utilisation de git et de Jekyll. Lisez notre [processus de documentation et de traduction](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), et [présentez-vous](https://github.com/jamulussoftware/jamulus/discussions) si vous souhaitez vous impliquer. Vous pouve ...
+msgstr ""
+"Nous utilisons [Weblate](https://hosted.weblate.org/projects/jamulus/"
+"#languages) pour traduire notre documentation. Lire à propos notre ["
```
The line breaks here look prone to error.  Is it correct to insert a line break after a `[` or within a url?

ann0see replied:
@trebmuh

trebmuh replied:
As far as I know, it's not a problem at all. It's not really "cut" and will be "re-glued" by whatever is processing the file after.
I mean, at least in a lot of other software I'm (or have been) translating, this is the same.
Edit : oh, I forget to add that this is Weblate which is doing so (I mean, it's not chosen by the translator).

ignotus666 commented:
That long list of files after "It also includes the following components:" is because they all share the same settings as Index-1 and it was easier (and is supposed to use fewer resources) when setting up the project - and I wasn't aware that it would cause this. The PR message is a bit excessive and misleading, it's as if Index-1 were the "parent" file of the project and any changes affect it when they don't, and every other file always gets listed even if unchanged. Giving each doc its own settings can be done though it'd be quite tedious. The PR message can be changed to just "Translation update from Weblate" or something like that but tell me what you think first.

I've also enabled squashing commits before they get sent here. I've set them to be squashed by author but that can be changed to per language/file or everything into one commit. We won't see if it worked until this PR's merged and the next one is opened though.

ignotus666 commented:
Ok, squashing is working.

ignotus666 commented:
Could we merge this now? I'd like to remove obsolete .po files but if I do so before this is merged there might be conflicts (mainly with "Directories/"Custom-Directories").

ignotus666 commented:
Also @ann0see  - what do you think about simplifying the PR message?

ann0see commented:
Yes sure. If the commit message contains the language and a short description, it should be ok.

ann0see commented:
> Could we merge this now?

Yes

ann0see commented:
I'd remove the "Hosted" part. 


Probably @hoffie has better insight in good commit messages as he's the person who introduced some style in the main repo.

trebmuh committed:
Translated using Weblate (French)

Currently translated at 99.2% (138 of 139 strings)

Translated using Weblate (French)

Currently translated at 100.0% (10 of 10 strings)

Co-authored-by: Olivier Humbert <trebmuh@tuxfamily.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/contribution/fr/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/fr/
Translation: Jamulus/Contribution
Translation: Jamulus/Running-a-Server

ignotus666 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (25 of 25 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (27 of 27 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (15 of 15 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (17 of 17 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (135 of 135 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (36 of 36 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (17 of 17 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (47 of 47 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (135 of 135 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (139 of 139 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (36 of 36 strings)

Translated using Weblate (Spanish)

Currently translated at 99.2% (138 of 139 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (19 of 19 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (27 of 27 strings)

Co-authored-by: ignotus <darylo1@hotmail.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/contribution/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/faq/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/general/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/include-server-commands/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/include-shared-commands/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/privacy-statement/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/es/
Translation: Jamulus/1-index
Translation: Jamulus/Contribution
Translation: Jamulus/Directories
Translation: Jamulus/FAQ
Translation: Jamulus/Include-Server-Commands
Translation: Jamulus/Include-Shared-Commands
Translation: Jamulus/Privacy-Statement
Translation: Jamulus/Running-a-Server
Translation: Jamulus/Software-Manual
Translation: Jamulus/Tips-Tricks-More
Translation: Jamulus/general

jujudusud committed:
Translated using Weblate (French)

Currently translated at 100.0% (15 of 15 strings)

Translated using Weblate (French)

Currently translated at 100.0% (28 of 28 strings)

Translated using Weblate (French)

Currently translated at 100.0% (15 of 15 strings)

Co-authored-by: Julien Taverna <jujudusud@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/custom-directories/fr/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/fr/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-troubleshooting/fr/
Translation: Jamulus/Custom-Directories
Translation: Jamulus/Directories
Translation: Jamulus/Server-Troubleshooting

ann0see commented:
@ignotus666 you need to translate directoryfile and listfilter probably again. after I resolved the conflicts

ann0see committed:
Merge branch 'next-release' into weblate-jamulus-1-index

ann0see commented:
I think fixing conflicts on the weblate repo is more difficult. I now did it on GitHub the standard way.
